run-prod.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset -o pipefail
+
+readonly repository="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
+export COMPOSE_PROJECT_NAME="tarpisto-prod"
+source "$repository/scripts/common.sh"
+
+function stop() {
+  pushd "$repository"
+  required_command docker
+  required_command docker-compose
+  docker-compose -f docker-compose.yml -f docker-compose.prod.yml down -v || true
+  popd
+}
+trap stop EXIT
+
+function main() {
+    required_command npm
+    required_command docker
+    required_command docker-compose
+
+    pushd "$repository"
+
+    export PORT=${PORT:-"9000"} # Docker-compose exposes this port in 9020
+
+    export USER_SERVICE_SERVICE_ID=${USER_SERVICE_SERVICE_ID:-"11188b9c-9534-4faf-8355-60973b720647"}
+    export USER_SERVICE_URL=${USER_SERVICE_URL:-"http://127.0.0.1:8080"}
+    export USER_SERVICE_SECRET=${USER_SERVICE_SECRET:-"catlike-meringue-tying-PASTERN-bed-simply"}
+
+    export NEXTAUTH_URL=${NEXTAUTH_URL:-"http://127.0.0.1:9000"}
+    export NEXTAUTH_SECRET=${NEXTAUTH_SECRET:-"catlike-meringue-tying-PASTERN-bed-simply"}
+
+    export AWS_REGION=${AWS_REGION:-"eu-west-1"}
+    export AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID:-"tarpisto"}
+    export AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY:-"tarpisto"}
```

Comment:
Same here as above. No fallback secrets on prod.

Reply:
Reworked env values fetching and scripts now will tell in which kind of env them are run on. Now it's more clear that prod gets only NODE_ENV, APP_ENV and PORT by default with prod-safe values. Next will check on startup if needed values are missing.